.github/workflows/workflow.yml

```diff
@@ -31,9 +31,9 @@ jobs:
         chmod 600 spec/fixtures/.netrc
     - name: Install dependencies
       run: |
-        gem install bundler
-        gem update bundler
```

Comment:
`gem install bundler` installs the latest Bundler, so `gem update bundler` seems unnecessary.